packages/react-components/react-field/src/stories/SwitchField/SwitchFieldDescription.md

```diff
@@ -0,0 +1 @@
+SwitchField adds a Label, validation message, and hint text to the Switch control.
```

Comment:
Should this be like `CheckboxField` where a `Label` is not mentioned since `Switch` already has one?

```suggestion
SwitchField adds a validation message and hint text to the Switch control.
```

Reply:
This behavior is pending discussion with design. I'll update it once we decide how SwitchField should work.